### PR TITLE
sig-packaging

### PR DESCRIPTION
**Pull request by calvinrp**
Draft proposal for `SIG-Registries` -> `SIG-Packaging`

### CODE REVIEW

File: SIGs/sig-packaging/proposal.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+under the auspices of the TSC of the Bytecode Alliance, as specified in the TSC’s charter. 
+
+SIG-Packaging's primary goal is to describe and build packaging tools that improve the
+developer distribution experience, for both consumers and publishers, of WebAssembly components.
+The tools should help guest and host toolchains integrate WebAssembly components for consuming and
+publishing Wasm components. The group should seek to further registry designs
+but also create integrations and coordinate with other related packaging efforts.
+
+Key deliverables and areas of focus:
+- Development of `wkg` and `wasm-pkg-tools` for consuming and publishing components to OCI and Warg registries;
```
Comment (lann) [inline SIGs/sig-packaging/proposal.md]:
```suggestion
- Development of shared libraries and common tooling for consuming and publishing components to OCI and Warg registries;
```

**Comment by ricochet** [comment]
Thank you for your effort towards transparency to reflect the SIG's current scope and efforts. 

This is essentially a reboot of the SIG. We in the TSC agree that it makes sense for the reboot to reflect the founding members listed in this document.

Because this is a reboot, we would like for this SIG to consider onboarding with existing SIG processes (now that we have them). This includes adding the SIG meeting to the Bytecode Alliance events calendar for public visibility, onboarding to password management (if needed), and management via the bytecodealliance.org google workspace for groups. Please work with @disquisitioner for assistance.

The TSC has reviewed this project proposal and find it in order.

**Comment by calvinrp** [comment]
Great! @ricochet merge the PR?

**Comment by ricochet** [comment]
Once the reboot has completed by archiving or renaming any necessary repos, onboarding to existing tools, etc, we'll close the issue.

**Pull request closed without merging**